matrix_sdk/src/async_client.rs

```diff
@@ -447,6 +446,45 @@ impl AsyncClient {
         self.send(request).await
     }
 
+    /// Forget a room by `RoomId`.
+    ///
+    /// Returns a `forget_room::Response`, an empty response.
+    ///
+    /// # Arguments
+    ///
+    /// * room_id - The `RoomId` of the room to be forget.
```

Comment:
The arguments should be inside an inline code block like it's shown [here](https://doc.rust-lang.org/stable/rust-by-example/meta/doc.html).